alvr/xtask/src/packaging.rs

```diff
@@ -103,8 +103,6 @@ pub fn package_streamer(
 ) {
     let sh = Shell::new().unwrap();
 
-    fs::remove_dir_all(afs::streamer_build_dir()).ok();
-
```

Comment:
What's the rationale behind removing it now?

Reply:
I realized we were already calling it in build launcher